src/Http/Controllers/AutoDocController.php

```diff
@@ -18,9 +19,47 @@ public function __construct()
 
     public function documentation()
     {
-        $documentation = $this->service->getDocFileContent();
+        $this->documentation = json_decode(json_encode($this->service->getDocFileContent()), true);
 
-        return response()->json($documentation);
+        $this->mergingContentsDocFiles();
+
+        return response()->json($this->documentation);
+    }
+
+    public function mergingContentsDocFiles()
+    {
+        $additionalDocs = config('auto-doc.additional_paths');
```

Comment:
```suggestion
        $additionalDocs = config('auto-doc.additional_paths', []);
```

with this suggestion we can remove next check for non additional pathes empty